cores/arduino/stm32/timer.h

```diff
@@ -58,7 +58,8 @@ extern "C" {
 #define TIM1_IRQn TIM1_UP_TIM10_IRQn
 #define TIM1_IRQHandler TIM1_UP_TIM10_IRQHandler
 #endif
-#elif defined(STM32H7xx) || defined(STM32L5xx) || defined(STM32MP1xx) || defined(STM32WLxx)
+#elif defined(STM32H7xx) || defined(STM32L5xx) || defined(STM32MP1xx) ||\
```

Comment:
I approve this PR, but I wonder whether we could simplify such complex #if which depends on stm32serie,
and replace it with something more generic like:
#elif defined(TIM1_UP_IRQn)

Reply:
Unfortunately no because `TIM1_UP_IRQn` is an enum value so it can not be preprocessed.